docs/UserGuide.md

```diff
@@ -149,7 +157,8 @@ You can export existing patient data to a csv file stored in a specific location
 
 Format: `export`
 
-- Please create a directory named "RapidTracerData" and create a new csv file named "PatientData" in the directory.
+- Create a directory named `RapidTracerData` within the file where RapidTracer is launched.
+- Create a new csv file named `PatientData.csv` in the `RapidTracerData` directory.
```

Comment:
Is it possible for the application to automatically generate the directory if it doesn't exist?

Reply:
Actually based on the createDirectory() method in the exportCommand class, the directory and file will be auto generated if they don't exist previously.

Reply:
have added one remark in the UG to notify the user that if they don't create the directory and file as stated, they will be auto generated.